01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -20,7 +20,7 @@ function __construct($name, $age)
     }
 
     /**
-     * @param int $age
+     * @param $age int
```

Comment:
I aggree, and I always write `@param` PHPDoc comments à la _type, variable name, description_. However, all other comments throughout the workshop materials are in the manner stated in my commit.